SCF-Session-8/zooManagement/ZooTest.java

```diff
@@ -0,0 +1,114 @@
+package zooManagement;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class ZooTest
+{
+	Zoo zoo = new Zoo();
```

Comment:
Test method are covering only positive cases. Why negative cases are not covered?